playbook/app/pb_kits/playbook/pb_phone_number_input/_phone_number_input.tsx

```diff
@@ -87,39 +99,84 @@ const PhoneNumberInput = (props: PhoneNumberInputProps) => {
   const inputRef = useRef<HTMLInputElement>()
   const [inputValue, setInputValue] = useState(value)
   const [itiInit, setItiInit] = useState<any>()
-  const [error, setError] = useState('')
+  const [error, setError] = useState(props.error)
   const [dropDownIsOpen, setDropDownIsOpen] = useState(false)
   const [selectedData, setSelectedData] = useState()
 
+  useEffect(() => {
+    if (error?.length > 0) {
+      onValidate(false)
+    } else {
+      onValidate(true)
+    }
+  }, [error, onValidate])
+
+  if (itiInit) isValid(itiInit.isValidNumber())
```

Comment:
I wonder if this is being continuously called and may be causing the issue with the onboarding queue. Is it possible to run this only in an onChange? Just for reference the issue is that the page won't render. You can check it out using [this](https://pr32458.nitro-web.beta.hq.powerapp.cloud/human_resources/on_boarding/a5a3b362-6d40-46fd-a7bd-f92d037af09d) link. @thestephenmarshall

Reply:
@kellyeryan Stephen is out of office this week, so i'll get another engineer to take a look today, and we will get back to you as soon as possible.

Reply:
@kellyeryan just commenting here for visibility.

Could we see a branch/PR? We would like to check what is being passed into the `isValid` prop.

Reply:
@kellyeryan after you test, could you let us know if we can resolve this?
  - [Link to fake candidate](https://pr32645.nitro-web.beta.gm.powerapp.cloud/human_resources/on_boarding/e806124c-56fc-45f9-ac49-b9f7e50cc93c)
  - [Link to Alpha PR](https://github.com/powerhome/nitro-web/pull/32645)

cc @jasperfurniss

## Details

See fix [in this commit](https://github.com/powerhome/playbook/pull/2582/commits/539aebb72b2d36e54573976d20e4a33afde5288a). 

To document the issue:
  - Human Resources has their own `PhoneNumberInput` that is an extension of the Playbook kit
  - HR's kit uses a [third-party library's custom validation](https://github.com/powerhome/nitro-web/blob/master/components/human_resources/app/components/FormInputs/PhoneNumberInput.jsx#L20-L28) which gets passed into PB's `isValid` prop
  - [Prior to the fix](https://github.com/powerhome/playbook/pull/2582/commits/539aebb72b2d36e54573976d20e4a33afde5288a#diff-fe537e3535d87cd96eba5d374fe585ff02d5328eeab49126e2a4f46fd0b53bc2L114), `isValid` was being called before the page loaded. In HR's case, there was no form because the page has not loaded, and `addErrors()` was trying add or clear errors.
  - With this fix, `isValid` will be called under these scenarios:
    1. After the user enters a number in the field
    2. When the input box loses focus

Reply:
Thanks for all the hard work! Looks good to me! Way to go!